src/sentry/static/sentry/app/components/commitAuthorStats.jsx

```diff
@@ -113,35 +104,32 @@ const CommitAuthorStats = createReactClass({
 
     return (
       <div style={{marginTop: 5}}>
-        <h6 className="nav-header m-b-1">Commits by Author</h6>
+        <h6 className="nav-header m-b-1">{t('Commits by Author')}</h6>
         {!commitAuthorValues.length && this.renderEmpty()}
-        <ul className="list-group">
-          {commitAuthorValues.map((commitAuthor, i) => {
-            let {author, commitCount} = commitAuthor;
-            return (
-              <li
-                key={i}
-                className="list-group-item list-group-item-sm list-group-avatar"
-              >
-                <div className="row row-flex row-center-vertically">
-                  <div className="col-sm-8">
-                    <span
-                      className="avatar-grid-item m-b-0 tip"
-                      title={author.name + ' ' + author.email}
-                    >
+        <Panel>
+          <PanelBody>
+            {commitAuthorValues.map((commitAuthor, i) => {
+              let {author, commitCount} = commitAuthor;
+              return (
+                <PanelItem key={i} p={1} align="center">
+                  <Flex>
```

Comment:
Does this need to be `Flex`? -or- if we were porting, should have `align="center"`.

Reply:
benefit of `Flex` is that it seems to be already aligning center by default